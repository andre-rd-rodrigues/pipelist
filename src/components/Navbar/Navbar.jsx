import React from "react";
import { Container, Navbar } from "react-bootstrap";
import styles from "./navbar.module.scss";

const AppNavbar = () => {
  return (
    <nav
      className={`${styles.navbar} navbar navbar-expand-lg navbar-light fixed-top`}
    >
      <Container>
        <Navbar.Brand>
          <img
            src={`${process.env.PUBLIC_URL}/images/logo.png`}
            alt="Pipedrive"
          />
        </Navbar.Brand>
      </Container>
    </nav>
  );
};

export default AppNavbar;
