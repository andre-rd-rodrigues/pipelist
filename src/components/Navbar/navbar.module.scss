@import "styles/mixins";

.navbar {
  background-color: #26292c;
  padding: 0.8rem;
  & img {
    max-width: 10rem;
  }
}
