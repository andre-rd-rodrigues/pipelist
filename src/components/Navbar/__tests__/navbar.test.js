import React from "react";
import renderer from "react-test-renderer";
import Navbar from "../Navbar";

describe("Navbar", () => {
  it("should render correctly", () => {
    const wrapper = renderer.create(<Navbar />);
    expect(wrapper).toMatchSnapshot();
  });
});
