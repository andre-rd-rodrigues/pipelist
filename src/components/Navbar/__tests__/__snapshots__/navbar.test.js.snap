// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render correctly 1`] = `
<nav
  className="navbar navbar navbar-expand-lg fixed-top"
>
  <span
    className="navbar-brand"
  >
    <img
      alt="Pipedrive"
      src="/images/logo.png"
    />
  </span>
</nav>
`;
